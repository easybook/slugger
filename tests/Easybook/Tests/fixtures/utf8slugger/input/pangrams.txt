Bulgarian: Жълтата дюля беше щастлива, че пухът, който цъфна, замръзна като гьон.
Croatian: Gojazni đačić s biciklom drži hmelj i finu vatu u džepu nošnje.
Czech: Příliš žluťoučký kůň úpěl ďábelské kódy.
Danish: Høj bly gom vandt fræk sexquiz på wc.
English: The quick brown fox jumps over the lazy dog.
Esperanto: Eble ĉiu kvazaŭ-deca fuŝĥoraĵo ĝojigos homtipon.
French: Les naïfs ægithales hâtifs pondant à Noël où il gèle sont sûrs d'être déçus en voyant leurs drôles d'œufs abîmés.
Finnish: On sangen hauskaa, että polkupyörä on maanteiden jokapäiväinen ilmiö.
German: Falsches Üben von Xylophonmusik quält jeden größeren Zwerg.
Greek: Ταχίστη αλώπηξ βαφής ψημένη γη, δρασκελίζει υπέρ νωθρού κυνός
Hungarian: Árvíztűrő tükörfúrógép.
Icelandic: Sævör grét áðan því úlpan var ónýt.
Latvian: Četri psihi faķīri vēlu vakarā zāģēja guļbūvei durvis, fonā šņācot mežam.
Lithuanian: Įlinkdama fechtuotojo špaga sublykčiojusi pragręžė apvalų arbūzą .
Maltese: Jekk jogħġbok ibgħat lili xi pangram (ĊċĠġĦħŻż).
Norwegian: Vår sære Zulu fra badeøya spilte jo whist og quickstep i min taxi.
Polish: Pchnąć w tę łódź jeża lub osiem skrzyń fig.
Portuguese: Luís argüia à Júlia que «brações, fé, chá, óxido, pôr, zângão» eram palavras do português.
Romanian: Muzicologă în bej vând whisky și tequila, preț fix.
Russian: Съешь же ещё этих мягких французских булок да выпей чаю.
Sami: Vuol Ruoŧa geđggiid leat máŋga luosa ja čuovžža.
Slovak: Starý kôň na hŕbe kníh žuje tíško povädnuté ruže, na stĺpe sa ďateľ učí kvákať novú ódu o živote.
Spanish: Benjamín pidió una bebida de kiwi y fresa. Noé, sin vergüenza, la más exquisita champaña del menú.
Swedish: Flygande bäckasiner söka strax hwila på mjuka tuvor.
Turkish: Pijamalı hasta yağız şoföre çabucak güvendi.
Ukrainian: Чуєш їх, доцю, га? Кумедна ж ти, прощайся без ґольфів!.