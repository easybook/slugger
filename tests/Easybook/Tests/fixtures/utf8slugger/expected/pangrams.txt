bulgarian-zltata-dula-bese-sastliva-ce-puht-koito-cfna-zamrzna-kato-gon
croatian-gojazni-dacic-s-biciklom-drzi-hmelj-i-finu-vatu-u-dzepu-nosnje
czech-prilis-zlutoucky-kun-upel-dabelske-kody
danish-hoj-bly-gom-vandt-fraek-sexquiz-pa-wc
english-the-quick-brown-fox-jumps-over-the-lazy-dog
esperanto-eble-ciu-kvazau-deca-fushorajo-gojigos-homtipon
french-les-naifs-aegithales-hatifs-pondant-a-noel-ou-il-gele-sont-surs-detre-decus-en-voyant-leurs-droles-doeufs-abimes
finnish-on-sangen-hauskaa-etta-polkupyora-on-maanteiden-jokapaivainen-ilmio
german-falsches-uben-von-xylophonmusik-qualt-jeden-grosseren-zwerg
greek-tachiste-alopex-baphes-psemene-ge-draskelizei-yper-nothrou-kynos
hungarian-arvizturo-tukorfurogep
icelandic-saevor-gret-adan-thvi-ulpan-var-onyt
latvian-cetri-psihi-fakiri-velu-vakara-zageja-gulbuvei-durvis-fona-snacot-mezam
lithuanian-ilinkdama-fechtuotojo-spaga-sublykciojusi-pragreze-apvalu-arbuza
maltese-jekk-joghgbok-ibghat-lili-xi-pangram-ccgghhzz
norwegian-var-saere-zulu-fra-badeoya-spilte-jo-whist-og-quickstep-i-min-taxi
polish-pchnac-w-te-lodz-jeza-lub-osiem-skrzyn-fig
portuguese-luis-arguia-a-julia-que-bracoes-fe-cha-oxido-por-zangao-eram-palavras-do-portugues
romanian-muzicologa-in-bej-vand-whisky-si-tequila-pret-fix
russian-ses-ze-ese-etih-magkih-francuzskih-bulok-da-vypei-cau
sami-vuol-ruota-gedggiid-leat-manga-luosa-ja-cuovzza
slovak-stary-kon-na-hrbe-knih-zuje-tisko-povadnute-ruze-na-stlpe-sa-datel-uci-kvakat-novu-odu-o-zivote
spanish-benjamin-pidio-una-bebida-de-kiwi-y-fresa-noe-sin-verguenza-la-mas-exquisita-champana-del-menu
swedish-flygande-backasiner-soka-strax-hwila-pa-mjuka-tuvor
turkish-pijamali-hasta-yagiz-sofore-cabucak-guvendi
ukrainian-cues-ih-docu-ga-kumedna-z-ti-prosaisa-bez-golfiv